app/assets/stylesheets/components/_footer.scss

```diff
@@ -1,14 +1,15 @@
-
 .footer {
-  background: #F4F4F4;
+  background: #f4f4f4;
   display: flex;
   align-items: center;
   justify-content: space-between;
   height: 64px;
   padding: 0px 15px;
-  color: rgba(0,0,0,0.3);
+  color: rgba(0, 0, 0, 0.3);
   // flex-wrap: wrap;
   font-size: 0.8rem;
+  position: absolute;
```

Comment:
avoid position absolute as much as possible.

Reply:
I teach 3 ways of sticking the footer to the bottom without position absolute [here](https://github.com/andrerferrer/quickTips/blob/master/CSS/Footer.md)

@khamzahabibullah @Sajakhtar

Reply:
Thanks. Done!